lib/membrane/rtp/session_bin.ex

```diff
@@ -634,4 +638,27 @@ defmodule Membrane.RTP.SessionBin do
 
     {rtp_extensions, maybe_link_twcc, state}
   end
+
+  defp maybe_handle_twcc_sender(pad_ssrc, ctx) do
+    # Workaround: as TWCC is a transport-wide extension, there should exist only one TWCC sender
+    # child that handles packets for all outgoing streams. As there is no support for declaring
+    # outbound extensions, outbound TWCC will be enabled if inbound TWCC has been enabled.
+    should_link? = Map.has_key?(ctx.children, :twcc_receiver)
+    should_create_child? = not Map.has_key?(ctx.children, :twcc_sender)
+
+    if should_link? do
+      &(&1
```

Comment:
can you change it to `fn` in this if path? Would be more readable than it is now.